magento/pwa-studio
# [feature] Pass rest attributes to radio group items

## Description

Forwarding additional attributes to Radio component using RadioGroup.

## Related Issue
Closes #2270.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps
1. Use the RadioGroup on a page.
2. Pass other attributes in the item object like this:
```
<RadioGroup items={[
    {
        value: 'value 1',
        label: 'label 1',
        disabled: true
    },
    {
        value: 'value 2',
        label: 'label 2',
        role: 'radio'
    }
]}/>
```
3. Check the new attributes rendered in the page. The first should be disabled the second should be enabled.

## Screenshots / Screen Captures

![image](https://user-images.githubusercontent.com/21162174/78467313-b2573400-76e1-11ea-832c-377ea4d7fb30.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
